perfzero/lib/cloud_manager.py

```diff
@@ -133,9 +134,37 @@ def create(username, project, zone, machine_type, accelerator_count,
   logging.info('Successfully created gcloud computing instance %s with %s accelerator.\n',  # pylint: disable=line-too-long
                instance_name, accelerator_count)
 
+  # Wait until we can ssh to the newly created computing instance
+  cmd = 'gcloud compute ssh {} --project={} --zone={} --strict-host-key-checking=no --command="ls"'.format(instance_name, project, zone)  # pylint: disable=line-too-long
+  ssh_remaining_retries = 12
+  ssh_error = None
+  while ssh_remaining_retries > 0:
+    ssh_remaining_retries -= 1
+    try:
+      run_command(cmd, is_from_user=False)
+      ssh_error = None
+    except Exception as error:  # pylint: disable=W0703
+      ssh_error = error
+      if ssh_remaining_retries:
+        logging.info('Cannot ssh to the computing instance. Try again after 5 seconds')  # pylint: disable=line-too-long
```

Comment:
Nit: wrap strings instead of disabling lint.

Reply:
Sure. It is changed now. I will have a followup patch to change all existing code styles to wrap the strings as appropriate.